config/config.json

```diff
@@ -364,7 +376,7 @@
       "title": "ComputeManagementClient"
     },
     "azure_mgmt_compute_2019_04_01": {
-      "markdown": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/4e644fc9ee1cf774f68b01d6b6be4fdf7118809c/specification/compute/resource-manager/readme.md",
+      "markdown": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/9b375a5cc64ff46e77930c400085bb9a864c1d9f/specification/compute/resource-manager/readme.md",
```

Comment:
Do we need refresh other api versions as well?

Reply:
I only refreshed the api version used in the latest profile.